src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -4346,6 +4389,7 @@ void SmartScript::InitTimer(SmartScriptHolder& e)
             RecalcTimer(e, e.event.distance.repeat, e.event.distance.repeat);
             break;
         case SMART_EVENT_NEAR_UNIT:
+        case SMART_EVENT_NEAR_UNIT_NEGATION:
```

Comment:
I think this should have its own RecalcTimer, as you have e.event.nearUnitNegation.timer

Reply:
From what I understood this only takes the *position* of the field. SMART_EVENT_NEAR_PLAYERS is the same, for example. Same I think goes for SMART_EVENT_UPDATE and its derivatives